extensions/interactions/TextInput/text-input-prediction.service.spec.ts

```diff
@@ -23,45 +23,45 @@ import { PredictionResultObjectFactory } from
   'domain/classifier/PredictionResultObjectFactory';
 import { SVMPredictionService } from 'classifiers/svm-prediction.service';
 import { UpgradedServices } from 'services/UpgradedServices';
+import { TextInputPredictionService } from
+  'interactions/TextInput/text-input-prediction.service';
+import { TextInputTokenizer } from "classifiers/text-input.tokenizer";
 // ^^^ This block is to be removed.
 
-describe('Text Input Prediction Service', function() {
-  var $rootScope = null;
-  var $scope = null;
+describe('Text Input Prediction Service', () => {
+  let $rootScope = null;
+  let $scope = null;
 
   beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    $provide.value('CountVectorizerService', new CountVectorizerService());
-    $provide.value(
-      'PredictionResultObjectFactory', new PredictionResultObjectFactory());
-    $provide.value(
-      'SVMPredictionService', new SVMPredictionService(
-        new PredictionResultObjectFactory()));
-  }));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
+  beforeEach(angular.mock.module('oppia', ($provide) => {
+    const ugs = new UpgradedServices();
     for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
       $provide.value(key, value);
     }
   }));
 
-  describe('Test text prediction service', function() {
-    var predictionService;
-    beforeEach(angular.mock.inject(function($injector) {
-      predictionService = $injector.get('TextInputPredictionService');
+  describe('Test text prediction service', () => {
+    let predictionService: TextInputPredictionService;
+    beforeEach(() => {
+      predictionService = new TextInputPredictionService(
+        new CountVectorizerService(),
+        new SVMPredictionService(new PredictionResultObjectFactory()),
+        new TextInputTokenizer());
+    });
+    beforeEach(angular.mock.inject(($injector) => {
       $rootScope = $injector.get('$rootScope');
       $scope = $rootScope.$new();
     }));
-    it('should predict the same as oppia-ml', function() {
-      var classifierData =
-        window.__fixtures__['core/tests/data/text_input_classifier_data'];
-      var trainingData =
-        window.__fixtures__['core/tests/data/text_classifier_results'];
-      var correctPredictions = 0, totalAnswers = 0;
-      var predictedAnswerGroup = null;
+    it('should predict the same as oppia-ml', () => {
+      const classifierData =
+          window.__fixtures__['core/tests/data/text_input_classifier_data'];
+      const trainingData =
+          window.__fixtures__['core/tests/data/text_classifier_results'];
+      const correctPredictions = 0, totalAnswers = 0;
```

Comment:
Just curious -- Any reason for changing them to const?

Reply:
No specific reason.
I just thought it would be better since we are moving to arrow functions.

Reply:
Do we even need these variables? Do we use them somewhere?

Reply:
I crosschecked and removed the variables that were not used.